src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -4335,7 +4347,10 @@ public boolean accept(File pathname) {
             }
         }, true);
         copyFile(fileChanged, srcDir, tempConfig, targetFileName);
-        checkConfigFile(fileChanged, tempConfig);
+        // if feature generation is on, then it should succeed before installing features
+        if (!generateFeatures || generateFeaturesSuccess) {
+            checkConfigFile(fileChanged, tempConfig);
+        }
```

Comment:
May not be relevant given comments above, but it seems redundant to create the temporary configuration folder if we may be skipping `checkConfigFile`

Reply:
Added TODO to revisit this in the future based on the decision for when to skip install features.